Joinpoint Regression in R.R

```diff
@@ -61,133 +73,158 @@ ggplot(mortdata, aes(year, asdr, color = cod)) +
   theme_classic() +
   xlab("Year") +
   ylab("Age-Adjusted Death Rate (per 100,000)") +
-  tayloRswift::scale_color_taylor("lover") # yes, this IS a Github repo dedicated to color palettes of Taylor Swift albums
+  scale_color_taylor("lover")
 
-# And the same figure again with a log transformation (so we can better prepare for the following logistic analyses)
+# And the same figure again with a log transformation
+# (so we can better prepare for the following logistic analyses)
 
 mortdata$asdrlog <- log(mortdata$asdr)
-ggplot(mortdata, aes(year, asdrlog, color = cod)) +
+ggplot(mortdata, aes(year, asdr, color = cod)) +
```

Comment:
When looking at log data we want the scale of the y axis to be more interpretable so we are using ggplots built in logarithmic scale converter.